<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>新宋体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>TFTP CLIENT by XBA</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images/images.qrc">
    <normaloff>:/icon.png</normaloff>:/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>Basic Settings</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_8">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="ServerIPLabel">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Server IP:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ServerIPLineEdit">
             <property name="text">
              <string>127.0.0.1</string>
             </property>
             <property name="maxLength">
              <number>512</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="ServerPortLabel">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Server Port:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ServerPortLineEdit">
             <property name="focusPolicy">
              <enum>Qt::StrongFocus</enum>
             </property>
             <property name="text">
              <string>69</string>
             </property>
             <property name="maxLength">
              <number>128</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_7">
           <item>
            <widget class="QLabel" name="ModeLabel">
             <property name="text">
              <string>Mode:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ModeComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QLabel" name="LocalFileLabel">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Local File:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="RemoteFileLabel">
             <property name="frameShape">
              <enum>QFrame::NoFrame</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Raised</enum>
             </property>
             <property name="text">
              <string>Remote File:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLineEdit" name="LocalFileLineEdit">
               <property name="maxLength">
                <number>512</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="dirBtn">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="images/images.qrc">
                 <normaloff>:/openFile.png</normaloff>:/openFile.png</iconset>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="RemoteFileLineEdit">
             <property name="maxLength">
              <number>512</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="clearButtonEnabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Extended Settings</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QCheckBox" name="tsizeCheckBox">
             <property name="autoFillBackground">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>tsize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QCheckBox" name="blksizeCheckBox">
             <property name="text">
              <string>blksize</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QCheckBox" name="timeoutCheckBox">
             <property name="text">
              <string>timeout</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="tsizeLabel">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <property name="text">
              <string>Unknown</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="blksizeSpinBox">
             <property name="frame">
              <bool>true</bool>
             </property>
             <property name="maximum">
              <number>16384</number>
             </property>
             <property name="value">
              <number>512</number>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="timeoutSpinBox">
             <property name="maximum">
              <number>255</number>
             </property>
             <property name="value">
              <number>2</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="title">
         <string>Operate</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="GetBtn">
           <property name="minimumSize">
            <size>
             <width>30</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Get</string>
           </property>
           <property name="icon">
            <iconset resource="images/images.qrc">
             <normaloff>:/download.png</normaloff>:/download.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PutBtn">
           <property name="text">
            <string>Put</string>
           </property>
           <property name="icon">
            <iconset resource="images/images.qrc">
             <normaloff>:/upload.png</normaloff>:/upload.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="BreakBtn">
           <property name="text">
            <string>Break(Undone)</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="title">
       <string>Info</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2,1,1,2,1,15">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>↑:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="UploadLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>kB/s</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>↓:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="DownloadLabel">
           <property name="text">
            <string>0.00</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>kB/s</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QProgressBar" name="ProgressBar">
           <property name="value">
            <number>24</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="LogTextBrowser">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <property name="lineWidth">
          <number>1</number>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_10">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="ClearBtn">
           <property name="text">
            <string>Clear</string>
           </property>
           <property name="icon">
            <iconset resource="images/images.qrc">
             <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="SaveBtn">
           <property name="text">
            <string>Save</string>
           </property>
           <property name="icon">
            <iconset resource="images/images.qrc">
             <normaloff>:/save.png</normaloff>:/save.png</iconset>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="images/images.qrc"/>
 </resources>
 <connections/>
</ui>
